Example/Source/Examples/Layout/OffsetBrickViewController.swift

```diff
@@ -24,15 +24,15 @@ class OffsetBrickViewController: BaseRepeatBrickViewController {
 }
 
 extension OffsetBrickViewController: OffsetLayoutBehaviorDataSource {
-    func offsetLayoutBehavior(behavior: OffsetLayoutBehavior, originOffsetForItemAtIndexPath indexPath: NSIndexPath, withIdentifier identifier: String, inCollectionViewLayout collectionViewLayout: UICollectionViewLayout) -> CGSize? {
+    func offsetLayoutBehaviorWithOrigin(_ behavior: OffsetLayoutBehavior, originOffsetForItemAtIndexPath indexPath: IndexPath, withIdentifier identifier: String, inCollectionViewLayout collectionViewLayout: UICollectionViewLayout) -> CGSize? {
```

Comment:
The problem with that is that the compiler confuses it with the other function of the dataSource. We need different names for it. If we create a protocol with the same function name but different arguments, the compiler gets crazy